### PR TITLE
Add save, list, and get routes; update README

### PR DESCRIPTION
**Pull request by tkyi**

### CODE REVIEW

**Comment by jer** [comment]
👍

File: lib/get.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        tags: ['api', 'pipelines'],
+        handler: (request, reply) => {
+            const config = {
+                table: 'pipelines',
+                params: {
+                    id: request.params.id
+                }
+            };
+
+            datastore.get(config, reply);
```
Comment (nkatzman) [inline lib/get.js]: The only thing about get is that when the `{id}` doesn't exist the datastores are returning null. Which means that we might want to handle returning 404 in that case

Comment (d2lam) [inline lib/get.js]: Good catch Noah

**Comment by nkatzman** [comment]
👍

**Pull request merged**